lib/libutee/include/sdp_pa.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#ifndef __SDP_PA_H
+#define __SDP_PA_H
+
+#define PTA_SDP_PA_UUID { 0x54c82831, 0x0170, 0x487d, \
+	{ 0xb7, 0xe6, 0xe9, 0x30, 0xf4, 0xfd, 0xc5, 0x24 } }
+
+#define PTA_CMD_SDP_VIRT_TO_PHYS			0x12
```

Comment:
Could you add a short description of the API. I.e:
```
/*
 * PTA_CMD_SDP_VIRT_TO_PHYS - Get physical address for the SDP buffer memref
 *
 * param[0] (in memref) - SDP buffer memory reference
 * param[1] (out value) - Physical address (.a=32bit LBS, .b=32bit MSB)
 * param[2] unused
 * param[3] unused
 *
 * This command is supported only for client that are SDP trusted application.
 */
#define PTA_CMD_SDP_VIRT_TO_PHYS			0x12
```

Reply:
Yes, Thanks for your example.